op-devstack/dsl/el.go

```diff
@@ -106,3 +106,24 @@ func (el *elNode) waitForNextBlock(blocksFromNow uint64) eth.BlockRef {
 func (el *elNode) stackEL() stack.ELNode {
 	return el.inner
 }
+
+func (el *elNode) WaitForFinalization() eth.BlockRef {
+	// Get current block and wait for it to be finalized
+	currentBlock, err := el.inner.EthClient().InfoByLabel(el.ctx, eth.Finalized)
```

Comment:
Should this be `eth.Unsafe` instead?
```suggestion
	currentBlock, err := el.inner.EthClient().InfoByLabel(el.ctx, eth.Unsafe)
```